calico_node/filesystem/startup.py

```diff
@@ -261,7 +261,7 @@ def main():
     # This is required because the confd templates expect to be able to fill in
     #  the IP by fetching it from the environment.
     with open('startup.env', 'w') as f:
-        f.write("IP=" + ip)
+        f.write("IP=" + str(ip))
```

Comment:
True, last time I remember it being a concern was in libcalico when encapsulating etcd endpoints, which is obviously a much higher scale. Since this is oneshot, I agree